lib/ipmi_vita.c

```diff
@@ -39,136 +39,104 @@
+	{VITA_CMD_ADDRINFO, "usage: addrinfo [<FRU-ID>]"},
+	{VITA_CMD_ACTIVATE, "usage: activate <FRU-ID>"},
+	{VITA_CMD_DEACTIVATE, "usage: deactivate <FRU-ID>"},
+	{VITA_CMD_POLICY_GET, "usage: policy get <FRU-ID>"},
+	{VITA_CMD_POLICY_SET,
+	 "usage: policy set <FRU-ID> <MASK> <VALUE>\n"
+	 "    MASK:  [3] affect the Default-Activation-Locked Policy Bit\n"
+	 "           [2] affect the Commanded-Deactivation-Ignored Policy Bit\n"
+	 "           [1] affect the Deactivation-Locked Policy Bit\n"
+	 "           [0] affect the Activation-Locked Policy Bit\n"
+	 "    VALUE: [3] value for the Default-Activation-Locked Policy Bit\n"
+	 "           [2] value for the Commanded-Deactivation-Ignored Policy Bit\n"
+	 "           [1] value for the Deactivation-Locked Policy Bit\n"
+	 "           [0] value for the Activation-Locked Policy Bit"},
+	{VITA_CMD_LED_PROP, "usage: led prop <FRU-ID>"},
+	{VITA_CMD_LED_CAP, "usage: led cap <FRU-ID> <LED-ID"},
 	{
 		VITA_CMD_LED_GET,
 		"usage: led get <FRU-ID> <LED-ID",
 	},
-	{
-		VITA_CMD_LED_SET,
-		"usage: led set <FRU-ID> <LED-ID> <FUNCTION> <DURATION> <COLOR>\n"
-		"    <FRU-ID>\n"
-		"    <LED-ID>   0-0xFE:    Specified LED\n"
-		"               0xFF:      All LEDs under management control\n"
-		"    <FUNCTION> 0:       LED OFF override\n"
-		"               1 - 250: LED blinking override (off duration)\n"
-		"               251:     LED Lamp Test\n"
-		"               252:     LED restore to local control\n"
-		"               255:     LED ON override\n"
-		"    <DURATION> 1 - 127: LED Lamp Test / on duration\n"
-		"    <COLOR>    1:   BLUE\n"
-		"               2:   RED\n"
-		"               3:   GREEN\n"
-		"               4:   AMBER\n"
-		"               5:   ORANGE\n"
-		"               6:   WHITE\n"
-		"               0xE: do not change\n"
-		"               0xF: use default color"
-	},
-	{
-		VITA_CMD_UNKNOWN,
-		"Unknown command"
-	},
-	{ 0, NULL }
-};
+	{VITA_CMD_LED_SET,
+	 "usage: led set <FRU-ID> <LED-ID> <FUNCTION> <DURATION> <COLOR>\n"
+	 "    <FRU-ID>\n"
+	 "    <LED-ID>   0-0xFE:    Specified LED\n"
+	 "               0xFF:      All LEDs under management control\n"
+	 "    <FUNCTION> 0:       LED OFF override\n"
+	 "               1 - 250: LED blinking override (off duration)\n"
+	 "               251:     LED Lamp Test\n"
+	 "               252:     LED restore to local control\n"
+	 "               255:     LED ON override\n"
+	 "    <DURATION> 1 - 127: LED Lamp Test / on duration\n"
+	 "    <COLOR>    1:   BLUE\n"
+	 "               2:   RED\n"
+	 "               3:   GREEN\n"
+	 "               4:   AMBER\n"
+	 "               5:   ORANGE\n"
+	 "               6:   WHITE\n"
+	 "               0xE: do not change\n"
+	 "               0xF: use default color"},
+	{VITA_CMD_UNKNOWN, "Unknown command"},
+	{0, NULL}};
 
 /* check if VITA 46.11 is supported */
-uint8_t
-vita_discover(struct ipmi_intf *intf)
+uint8_t vita_discover(struct ipmi_intf *intf)
```

Comment:
Yeah, didn't see this (https://github.com/ipmitool/ipmitool/wiki/Coding-Standards#function-definition), as just about none of the code follows the standard -- I'll adjust the clang-format.

Reply:
Also, I'm using the clang-format for https://github.com/openbmc/pam-ipmi/blob/master/.clang-format -- some defaults for values not explicitly set. -- do you have a clang-format file I can use?  I don't believe humans should be formatting code.  I spent years having to format my own code and it can such a time sync.